examples/README.md

```diff
@@ -307,6 +307,8 @@ Example | File | Description
 `scale_factor_override` | [`window/scale_factor_override.rs`](./window/scale_factor_override.rs) | Illustrates how to customize the default window settings
 `transparent_window` | [`window/transparent_window.rs`](./window/transparent_window.rs) | Illustrates making the window transparent and hiding the window decoration
 `window_settings` | [`window/window_settings.rs`](./window/window_settings.rs) | Demonstrates customizing default window settings
+`resizing` | [`window/resizing.rs`](./window/resizing.rs) | Demonstrates changing window size at runtime. Also used to validate that `bevy` can handle arbitrarily small windows
+`minimising` | [`window/minimising.rs`](./window/minimising.rs) | Demonstrates minimising a window from within your game. Also used to validate that `bevy` can handle minimised windows
```

Comment:
```suggestion
`minimising` | [`window/minimising.rs`](./window/minimising.rs) | Demonstrates minimising a window from within your game.
```

Move removed note to module level comment.

Reply:
I'd probably like to see this moved to some sort of integration tests folder eventually, but we can address that later once headless windows are working.